db/models/product.test.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const db = require('APP/db')
+    , {Product} = db
+    , {expect} = require('chai')
+
+/* global describe it before afterEach */
+
+describe('The `Product` model', () => {
+  before('Await database sync', () => db.didSync)
+  afterEach('Clear the tables', () => db.truncate({ cascade: true }))
+
+  var product;
+  beforeEach(function(){
+    product = Product.build({
+      name: 'Mens Bike',
+      category: 'Mountain',
+      price: 1000,
+      images: [1, 2, 3],
+      color: 'Red',
+      size: 'Medium',
+      availability: true,
+      reviewStars: 4.7,
+      description: 'this bike rocks'
+    });
+  });
+
+  describe('Columns exist', () => {
```

Comment:
The columns exist test seems redundant when we are testing the properties below.  Can we take this describe block out.